django/db/backends/oracle/base.py

```diff
@@ -531,7 +531,7 @@ def _fix_for_params(self, query, params, unify_by_values=False):
             args = {k: ":%s" % k for k in params}
             query %= args
         elif unify_by_values and params:
-            # Handle params as a dict with unified query parameters by their
+            # Handle params as a sequence with unified query parameters by their
```

Comment:
As far as I'm aware the previous comment it correct as `params` is converted to a dict.

Reply:
I dropped this commit.